source/getting-started/concepts-terminology.markdown

```diff
@@ -0,0 +1,55 @@
+---
+title: "Concepts and terminology"
+description: "User explanation of Home Assistant basics"
+---
+Now you're in Home Assistant, let's go over important parts of Home Assistant and what they do.
+
+## Dashboards
+
+Customizable pages to display information connected to and available in Home Assistant. By default there are two Dashboards, Overview and Energy. The Overview dashboard is the first thing you see after the [onboarding process](/getting-started/onboarding). More details can be found [here](/dashboards).
+
+![Dashboard](/images/getting-started/lovelace.png)
+## Integrations
+
+Software that allows Home Assistant to connect to other software and platforms. For example, the Philips Hue integration allows Home Assistant to talk to the Hue Bridge and read data such as the state of your lights through to controlling the lights themselves.
+
+![Integrations](/images/getting-started/integrations.png)
+
+A full list of compatible integrations can be found [here](/integrations).
+
+Once an integration has been added, the hardware and/or data are represented in Home Assistant as [devices and entities](#devices--entities).
+
+## Devices & Entities
+
+Devices are a logical grouping for one or more entities. A device may represent a physical device and the device may have one or more sensors which appear as entities associated to the device. For example, a motion sensor is represented as a device and the motion may provide motion detection, temperature and light levels as entities. Entities have states such as on when motion is detected and off when there is no motion.
+
+Devices and entities can be used in dashboards to [automations](#automations).
+
+![Home Assistant Device](/images/getting-started/home-assistant-device.png)
+
+## Automations
+
+A set of repeatable actions that can be set up to run automatically. Automations are made of three key components:
+1. Triggers - event(s) that start an automation like when the sun sets.
```

Comment:
```suggestion
1. Triggers - event(s) that start an automation. For example, when the sun sets.
```
I know I'm being a pain in the ass. But using parallelism can improve readability.

Reply:
@c0ffeeca7 you're not a PIA and I appreciate the feedback you're doing 👍

Reply:
I used *like* instead of for example because I was trying to avoid repetition on point 2 where I use *For example* however, if you think it's better this way then I am happy to amend.